crates/sui/src/ethereum_client_commands.rs

```diff
@@ -0,0 +1,85 @@
+use clap::Subcommand;
+use serde::{Deserialize, Serialize};
+use sui_types::base_types::ObjectID;
+
+#[derive(Subcommand)]
+pub enum EthClientCommands {
+    /// Approve a TX with Eth contract for a given dWallet.
+    #[command(name = "verify-message")]
+    EthApproveMessage {
+        /// Object of a [EthDwalletCap].
+        #[clap(long)]
+        eth_dwallet_cap_id: ObjectID,
+        /// The Message (TX) to verify.
+        #[clap(long)]
+        message: String,
+        // todo(zeev): in the future we can fetch the dwallet ID.
+        /// The DWallet ID.
+        #[clap(long)]
+        dwallet_id: ObjectID,
+        /// Gas object for gas payment.
+        #[clap(long)]
+        gas: Option<ObjectID>,
+        /// Gas budget for this call
+        #[clap(long)]
+        gas_budget: u64,
+        /// Instead of executing the transaction, serialize the bcs bytes of the unsigned transaction data
+        /// (TransactionData) using base64 encoding, and print out the string.
+        #[clap(long, required = false)]
+        serialize_unsigned_transaction: bool,
+        /// Instead of executing the transaction, serialize the bcs bytes of the signed transaction data
+        /// (SenderSignedData) using base64 encoding, and print out the string.
```

Comment:
```suggestion
        /// Instead of executing the transaction, serialize the bcs bytes of
        /// the signed transaction data
        /// (SenderSignedData) using base64 encoding, and print out the string.
```